src/pagination/getData.jsx

```diff
@@ -0,0 +1,54 @@
+import React, {  useState } from 'react';
+import "./pagination.css"
+import PaginationDisplay from '../pagination/pagination';
+const GetData = ({ postsNumber, postsArray, contentToShow }) => {
```

Comment:
I'll not review this implementation now, because I expect that it will change significantly after integration with backend